mdx2/scaling.py

```
@@ -129,7 +129,31 @@ def Bz(self):
 
     @property
     def B(self):
-        raise NotImplementedError
+        Nx, Ny, Nz = self.shape
+        k, l, m = np.meshgrid(np.arange(Nx), np.arange(Ny), np.arange(Nz), indexing="ij")
```

Ruff (0.14.8)

Ambiguous variable name: `l`

(E741)